src/userfunc.c

```diff
@@ -320,6 +320,16 @@ get_function_args(
 		++p;
 	    char_u *argend = p;
 
+	    // object variable can use constructor
```

Comment:
Can you update this comment to "object variable this.<name> can be used only in a constructor"?